developers_chamber/slack_utils.py

```diff
@@ -0,0 +1,35 @@
+import subprocess
+
+from slack_sdk import WebClient
+
+
+def run_sh(cmd: str) -> list:
+    return [
+        line for line in subprocess.run(
+            cmd, shell=True, check=True, stdout=subprocess.PIPE
+        ).stdout.decode('utf-8').rstrip().split('\n')
+    ]
+
+
+def upload_new_migration(token, channel, diff_branch):
+    client = WebClient(token=token)
+    current_branch = run_sh('git branch --show-current')[0]
```

Comment:
The project is already using `GitPython` library, why aren't you using it?
Check https://github.com/druids/developers-chamber/blob/master/developers_chamber/qa/base.py#L41

Reply:
The code was originally supposed to be in another project and there GitPython wasn't. I fixed it.